### PR TITLE
updated version

### PR DESCRIPTION
**Pull request by mossbeard**
first full version of the connector.

### CODE REVIEW

File: src/q2g-con-hypercube-qvx/Logger.cs [hunk 1]
```diff
@@ -33,7 +33,7 @@ public class ConnectorLogger
         public static ConnectorLogger CreateLogger()
         {
             var connectorPath = Assembly.GetExecutingAssembly().Location;
-            if(connectorPath.EndsWith("\\QlikTableConnector\\QlikTableConnector.exe"))
+            if(connectorPath.EndsWith("\\q2gconhypercubeqvx\\q2gconhypercubeqvx.exe"))
```
Comment (konne) [inline src/q2g-con-hypercube-qvx/Logger.cs]: please add a .toLowerInvariant() before compare

File: src/q2g-con-hypercube-qvx/TableServer.cs [hunk 2]
```diff
@@ -28,179 +31,177 @@ public class TableServer : QvxServer
...
             {
                 QvDataContractResponse response;
-
+                AppInstance.LoadMemory();
+                connection.MParameters.TryGetValue("host", out string parmStr);
                 switch (method)
                 {
                     case "getVersion":
-                        response = new Info { qMessage = GitVersionInformation.InformationalVersion };
+                        response = new Info { qMessage = "1.0.0" };
```
Comment (konne) [inline src/q2g-con-hypercube-qvx/TableServer.cs]: why go back? I already has fixed that.

File: src/q2g-con-hypercube-qvx/qlikapp/QlikApp.cs [hunk 3]
```diff
@@ -40,14 +49,13 @@ public QlikAppSession FirstSession
         public QlikApp(string connectUri) : this(null, connectUri, null) { }
 
         public QlikApp(string appName, string connectUri, IQlikCredentials credentials)
-        {
+        {
```
Comment (konne) [inline src/q2g-con-hypercube-qvx/qlikapp/QlikApp.cs]: remove spaces

**Comment by mossbeard** [comment]
@konne fixed recomended issus

**Pull request merged**